Functions.Templates/Resources/Resources.resx

```diff
@@ -861,194 +852,62 @@
   </data>
-  <data name="QueueTriggerBash_description" xml:space="preserve">
-    <value>(Experimental) A bash function that will be run whenever a message is added to a specified Azure Queue Storage</value>
+  <data name="HttpPOSTCRUD_description" xml:space="preserve">
+    <value>A function that adds entities to a Storage table when it receives an HTTP request</value>
   </data>
-  <data name="QueueTriggerBatch_description" xml:space="preserve">
-    <value>(Experimental) A Batch function that will be run whenever a message is added to a specified Azure Queue Storage</value>
+  <data name="HttpPUTCRUD_description" xml:space="preserve">
+    <value>A function that updates entity in a Storage table when it receives an HTTP request</value>
   </data>
-  <data name="QueueTriggerCSharp_description" xml:space="preserve">
-    <value>A C# function that will be run whenever a message is added to a specified Azure Queue Storage</value>
+  <data name="HttpTrigger_description" xml:space="preserve">
+    <value>A function that will be run whenever it receives an HTTP request, responding based on data in the body or query string</value>
   </data>
-  <data name="QueueTriggerFSharp_description" xml:space="preserve">
-    <value>An F# function that will be run whenever a message is added to a specified Azure Queue Storage</value>
+  <data name="ImageResizer_description" xml:space="preserve">
+    <value>A function that creates resized images whenever a blob is added to a specified container</value>
   </data>
-  <data name="QueueTriggerNodeJS_description" xml:space="preserve">
-    <value>A JavaScript function that will be run whenever a message is added to a specified Azure Queue Storage</value>
+  <data name="ManualTrigger_description" xml:space="preserve">
+    <value>A function that is triggered manually via the portal "Run" button</value>
   </data>
-  <data name="QueueTriggerPHP_description" xml:space="preserve">
-    <value>(Experimental) A PHP function that will be run whenever a message is added to a specified Azure Queue Storage</value>
+  <data name="QueueTrigger_description" xml:space="preserve">
+    <value>A function that will be run whenever a message is added to a specified Azure Storage queue</value>
   </data>
-  <data name="QueueTriggerPowerShell_description" xml:space="preserve">
-    <value>(Preview) A PowerShell function that will be run whenever a message is added to a specified Azure Queue Storage</value>
+  <data name="SaasFileTrigger_description" xml:space="preserve">
+    <value>A function that will be run whenever a file is added to an external file store</value>
   </data>
-  <data name="QueueTriggerPython_description" xml:space="preserve">
-    <value>(Experimental) A Python function that will be run whenever a message is added to a specified Azure Queue Storage</value>
+  <data name="SaasTable_description" xml:space="preserve">
+    <value>A function that will be run whenever data is added to an external table store</value>

```

Comment:
For the ones where we are removing the (preview)/(experimental) label, where are we exposing that info now? The category?

Reply:
This will be shown when they create the function and will update depending on the language:


![image](https://user-images.githubusercontent.com/8560637/32197602-2131ddec-bd82-11e7-98c5-0d5c83177ac9.png)